codalab/apps/web/tasks.py

```diff
@@ -851,6 +855,32 @@ def do_chahub_retries(limit=None):
             instance.save(force_to_chahub=True)
 
 
+@task(queue='site-worker')
+def send_chahub_general_stats():
+    if settings.DATABASES.get('default').get('ENGINE') == 'django.db.backends.postgresql_psycopg2':
+        # Only Postgres supports 'distinct', so if we can use the database, if not use some Python Set magic
+        organizer_count = Competition.objects.all().distinct('creator').count()
+    else:
+        users_with_competitions = list(ClUser.objects.filter(competitioninfo_creator__isnull=False))
+        user_set = set(users_with_competitions)
+        # Only unique users that have competitions
+        organizer_count = len(user_set)
+    raw_data = {
```

Comment:
no need for this temp var

Reply:
Which variable? Are you saying do all of organizer_count stuff on one line?

Only reason I did not do this is for read-ability and not smashing 2-3 casts in one line